lzy/allocator/src/main/java/ai/lzy/allocator/alloc/AllocateVmAction.java

```diff
@@ -228,6 +234,49 @@ private StepResult allocateVm() {
         }
     }
 
+    private StepResult allocateMountPod() {
+        if (!allocationContext.mountConfig().isEnabled()) {
+            return StepResult.ALREADY_DONE;
+        }
+
+        if (mountPodAllocated) {
+            return StepResult.ALREADY_DONE;
+        }
+
+        log().info("{} Allocating mount pod for VM {}", logPrefix(), vm.vmId());
+        try {
+            mountHolder = allocationContext.mountHolderManager().allocateMountHolder(vm.spec());
+            mountPodAllocated = true;
+        } catch (KubernetesClientException e) {
+            log().error("{} Cannot allocate mount holder for vm {}: {}", logPrefix(), vm.vmId(), e.getMessage());
```

Comment:
already exists?

Reply:
`allocateMountHolder` already resolves it